### ISSUE TITLE
Not working on OSX

### ISSUE THREAD

**Issue by BatmiBoom** [abreumatheus/lapce-pyright#1]
Hey! First of all, thanks for this plugin!. In windows is working super nice, but i cannot make it work on OSX. The server is installed :

<img width="1438" alt="Screenshot 2022-12-20 at 11 25 46 AM" src="https://user-images.githubusercontent.com/47833971/208690696-74af48a3-c6ce-434b-9eb2-32f91c5487d8.png">

But it doesn't work:

<img width="1099" alt="Screenshot 2022-12-20 at 11 26 24 AM" src="https://user-images.githubusercontent.com/47833971/208690792-a26ea793-33f2-4ed3-a07b-cf0219a90181.png">

**Comment by abreumatheus** [comment]
Did you install pyright with pip or with npm?

**Comment by BatmiBoom** [comment]
With pip `pip install pyright`. Can it be something about lapce and the virtual environment? I'm using poetry. But I have installed in the global scope and inside poetry

**Comment by abreumatheus** [comment]
It's strange because i'm using it on OSX with poetry also. Tomorrow I'll try to reproduce it in another mac and update you here.

**Comment by nstirnemann7** [comment]
To be precise, is only installing the server and having it on path, and install the plugin right?

**Comment by abreumatheus** [comment]
@nstirnemann7 Yes. Should be it. I really tried here but couldn't reproduce this error. I just ran `pip install pyright` and it worked flawlessly. It's important to notice that pyright requires node.js to be installed, maybe this is the missing link.

Obs: I'm on node `18.12.1`.

**Comment by nstirnemann7** [comment]
Hey! I tested it with `18.12.1`. Is there a way to debug this? Perhaps initializing lapce with a flag or something like that?

**Comment by lordirah** [comment]
Am also getting same error on fedora. Installed pyright via npm

**Comment by RichardFevrier** [comment]
@abreumatheus would you mind help us? 🙂
It seems that we all have `pyright`/`pyright-langserver` in our path.
Using it from the shell it's working properly.
No errors or warnings are raised in `lapce`.
Is there a way to debug your plugin like @nstirnemann7 was asking?

Fedora 38 - Python 3.12.0 from pyenv - Node v18.18.2 from nodenv - Pyright 1.1.332 from npm - Lapce v0.2.8

**Comment by RichardFevrier** [comment]
Okay so after a little investigation I was able to get it working under linux.
The problem seems related to `flatpak` sandbox and non-bundled python environments like `pyenv` (potentially the same problem occurs with OSX app sandbox).
